include/bluetooth/gatt.h

```diff
@@ -1669,6 +1678,13 @@ struct bt_gatt_subscribe_params {
 #endif /* CONFIG_BT_GATT_AUTO_DISCOVER_CCC */
 	/** Subscribe value */
 	uint16_t value;
+#if defined(CONFIG_BT_SMP)
+	/** Minimum required security for received notification. Upon reception
+	 * host verifies if current security level meets minimum requirements
+	 * before passing it to application.
+	 */
```

Comment:
I would like us to  explicitly say that the about the notifications are discarded.

```suggestion
	/** Minimum required security for this subscription. Notifications and
	  *  indications received over a connection with a lower security level are
	  *  silently discarded.
	  */
```